apps/video/js/thumbnail_item.js

```diff
@@ -64,7 +64,7 @@ function ThumbnailItem(videoData) {
   function convertToDOM(htmlText) {
```

Comment:
Nit: `htmlText` would read better as `wrappedHTML` or some nam indicating it is not raw HTML.